mmv1/third_party/terraform/tests/resource_vmwareengine_cluster_test.go.erb

```diff
@@ -0,0 +1,179 @@
+<% autogen_exception -%>
+package google
+<% unless version == 'ga' -%>
+
+import (
+	"fmt"
+	"testing"
+	"strings"
+
+	"github.com/hashicorp/terraform-provider-google/google/acctest"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/resource"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/terraform"
+	"github.com/hashicorp/terraform-provider-google/google/tpgresource"
+	transport_tpg "github.com/hashicorp/terraform-provider-google/google/transport"
+)
+
+func TestAccVmwareengineCluster_vmwareEngineClusterUpdate(t *testing.T) {
+	t.Parallel()
+
+	context := map[string]interface{}{
+		"region":        	 "southamerica-west1", // using region with low node utilization.
+		"org_id":          acctest.GetTestOrgFromEnv(t),
+		"billing_account": acctest.GetTestBillingAccountFromEnv(t),
+		"random_suffix":   RandString(t, 10),
+	}
+
+  config := vmwareEngineClusterConfig(context)
```

Comment:
not a blocker, but FWIW we generally pass variables (like the number of nodes in the management cluster) to a function like this rather than formatting them in separately.

Reply:
Actually this would probably be worth tweaking - I believe this is why the missing test detector is having issues.

Reply:
Update this method to take param instead of returning template and formatting it here.